java/detekt-cli/Portfile

```diff
@@ -46,3 +46,7 @@ destroot {
     xinstall -m 755 ${filespath}/detekt ${destroot}${prefix}/bin
     reinplace "s|_PREFIX_|${prefix}|g" ${destroot}${prefix}/bin/detekt
 }
+
+livecheck.type      regex
+livecheck.url       ${homepage}/detekt/changelog.html
+livecheck.regex     (\\d+\\.\\d+\\.\\d+) - \\d+-\\d+-\\d+
```

Comment:
Is it not possible to fix this by just adjusting `github.livecheck.regex` ?

Reply:
Yes that works. Changed it.

Reply:
Thanks